tests/admin_checks/tests.py

```diff
@@ -54,6 +55,55 @@ def test_checks_are_performed(self):
             admin.site.unregister(Song)
             admin.sites.system_check_errors = []
 
+    @override_settings(
+        DEBUG=True,
+        INSTALLED_APPS=['django.contrib.admin']
```

Comment:
Add a trailing comma in kwargs so if more items are added later, we don't have to modify this line again.